### PR TITLE
Preserve sudo passed in config section

### PR DESCRIPTION
**Pull request by retr0h**
The sudo option was getting reset, and would not respect option set
in molecule.yml.

Fixes: #496

### CODE REVIEW

File: molecule/verifier/testinfra.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def execute(self):
             self._molecule.verifier_options)
         testinfra_options['env'] = ansible.env
         testinfra_options['debug'] = self._molecule.args.get('debug', False)
-        testinfra_options['sudo'] = self._molecule.args.get('sudo', False)
+        if self._molecule.args.get('sudo'):
+            testinfra_options['sudo'] = True
```
Comment (signed8bit) [inline molecule/verifier/testinfra.py]: Would this affect the debug flag in the same way?

Comment (retr0h) [inline molecule/verifier/testinfra.py]: I don't believe so, b/c the debug flag is never passed in as an option to testinfra.

Comment (signed8bit) [inline molecule/verifier/testinfra.py]: So this one is really just not a True/False? Meaning that 'sudo' will never be present in self._molecule.args with a value of False?

Comment (retr0h) [inline molecule/verifier/testinfra.py]: sudo in the context of `self._molecule.args` would apply here.

One could override testinfra to always set sudo, or simply supply on the command line.

**Comment by retr0h** [comment]
```
  syntax: commands succeeded
  py27-ansible19-unit: commands succeeded
  py27-ansible19-functional: commands succeeded
  py27-ansible20-unit: commands succeeded
  py27-ansible20-functional: commands succeeded
  py27-ansible21-unit: commands succeeded
  py27-ansible21-functional: commands succeeded
  doc: commands succeeded
  congratulations :)
```

**Pull request merged**